ArduPlane/commands_logic.cpp

```diff
@@ -131,20 +131,17 @@ bool Plane::start_command(const AP_Mission::Mission_Command& cmd)
         break;
 
     case MAV_CMD_DO_FENCE_ENABLE:
-#if GEOFENCE_ENABLED == ENABLED
-        if (cmd.p1 != 2) {
-            if (!geofence_set_enabled((bool) cmd.p1)) {
-                gcs().send_text(MAV_SEVERITY_WARNING, "Unable to set fence. Enabled state to %u", cmd.p1);
-            } else {
-                gcs().send_text(MAV_SEVERITY_INFO, "Set fence enabled state to %u", cmd.p1);
-            }
-        } else { //commanding to only disable floor
-            if (! geofence_set_floor_enabled(false)) {
-                gcs().send_text(MAV_SEVERITY_WARNING, "Unable to disable fence floor");
-            } else {
-                gcs().send_text(MAV_SEVERITY_WARNING, "Fence floor disabled");
-            }
-        }    
+#if AC_FENCE == ENABLED
+        if (cmd.p1 == 0) { // disable fence
+            plane.fence.enable(false);
+            gcs().send_text(MAV_SEVERITY_INFO, "Fence disabled");
+        } else if (cmd.p1 == 1) { // enable fence
+            plane.fence.enable(true);
+            gcs().send_text(MAV_SEVERITY_INFO, "Fence enabled");
+        } else if (cmd.p1 == 2) { // disable fence floor only
+            plane.fence.disable_floor();
+            gcs().send_text(MAV_SEVERITY_INFO, "Fence floor disabled");
+        }
```

Comment:
These changes are unrelated and should be removed.

Reply:
Yes, the ONLY ONE line I changed in commands_logic.cpp is that I intended to have added is 

gcs().send_text(MAV_SEVERITY_INFO, "ETA is %d seconds", (int)current_loc.get_ETA(cmd.content.location));

...When the new WP is set. I don't know why any other lines appear to be changed. I have not made any other changes than this 1 line. (Maybe I have accidentally inserted or deleted a blank / empty line?)